org-roam.el

```diff
@@ -1025,6 +1025,37 @@ Otherwise, behave as if called interactively."
             (while (re-search-forward named-regex nil t)
               (replace-match (format "[[file:%s][\\1]]" relative-path))))
           (org-roam-db--update-file file-from)))
+      (unless (string= (file-name-directory path)
```

Comment:
I feel we should split this advice up into multiple functions.

Reply:
Yeah, same, but I figured we'd do it once we implement links to headlines.

Reply:
let me pull this down